content/_data/changelogs/weekly.yml

```diff
@@ -11320,13 +11328,17 @@
         - timja
       message: |-
         Bump sshd-core from 2.5.1 to 2.7.0 in Jenkins CLI.
-    - type: rfe
+    - type: major rfe
```

Comment:
major rfe should go first in the list I believe

I think category developer can be removed too? 

but other's may have different opinions

Reply:
Sure, I thought the same thing but due to the developer category, I placed it at last.
Do you want me to remove the developer cat. and place it at the top now, or should we wait for others' opinions?

Reply:
I think just remove developer and place at the top